stylesheet.css

```diff
@@ -0,0 +1,45 @@
+#main {
+  background-color: rgb(238, 238, 238);
+  border-radius: 2px;
+}
+
+body {
+  text-align: center;
+  font-size: 100%;
+  font-family: apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+}
+
+ul {
+  max-width: 550px;
+  margin: auto;
+  text-align: left;
+}
+
+#image {
+  margin: auto;
+  font-size: 100%;
+  max-width: 97%;
+  display: block;
+  padding: 7px 17px;
+}
+
+#img-caption {
+  background-color: white;
+  text-align: center;
```

Comment:
This is also a redundant code, the `text-align` property will be inherited from the `body` style as it's a descendant of the `body` element.